forgerock-auth/src/main/java/org/forgerock/android/auth/interceptor/IdentityGatewayAdviceInterceptor.java

```diff
@@ -16,6 +16,8 @@
 import org.jetbrains.annotations.NotNull;
 
 import java.io.IOException;
+import java.net.HttpURLConnection;
```

Comment:
Update copyright

Reply:
This file should not be here, rollback changes.